src/config.rs

```diff
@@ -285,16 +285,44 @@ impl Config {
             .ok_or_else(|| format_err!("An organization slug is required (provide with --org)"))
     }
 
+    /// Given a match object from clap, this returns the release from it.
+    pub fn get_release(&self, matches: &ArgMatches) -> Result<String> {
+        matches
+            .value_of("release")
+            .map(str::to_owned)
+            .or_else(|| env::var("SENTRY_RELEASE").ok())
```

Comment:
nit: why do we convert to `Option` with `ok()`? Can't we just call `or_else` on the `Result` directly?

Reply:
`map`, and `or_else` return type is `Option`, which forces us to convert.